.github/workflows/test-engine-package.yml

```
@@ -81,5 +83,13 @@ jobs:
         if: startsWith(matrix.target, 'linux') || startsWith(matrix.target, 'macos')
         run: chmod +x dist/run
 
+      # ref: https://github.com/VOICEVOX/voicevox_engine/issues/839
+      # ref: https://github.com/VOICEVOX/voicevox_engine/pull/806
+      - name: <Setup> Install libsndfile1 for Linux arm64 architecture
+        if: startsWith(matrix.target, 'linux') && endsWith(matrix.target, 'arm64')
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libsndfile1
```

`soundfile`、いつの間にかarm64に対応していました！！！
https://github.com/bastibe/python-soundfile/releases/tag/0.13.0

ということでissue作ってみました！このissueが達成されたらこちらも要変更かもです！

- https://github.com/VOICEVOX/voicevox_engine/issues/1516

---

あ、これ１つ目の方を↑のissueに切り替えても良さそう！
ということでちょっと勝手ながら変えさせていただきます！

```suggestion
      # ref: https://github.com/VOICEVOX/voicevox_engine/pull/806
      # ref: https://github.com/VOICEVOX/voicevox_engine/issues/1516
      - name: <Setup> Install libsndfile1 for Linux arm64 architecture
        if: startsWith(matrix.target, 'linux') && endsWith(matrix.target, 'arm64')
        run: |
          sudo apt-get update
          sudo apt-get install -y libsndfile1
```